src/app/shared/learning-object/learning-object.component.html

```diff
@@ -19,7 +19,10 @@
           <span *ngFor="let i of learningObject.contributors.slice(0,2)">
             , {{ i.name }}
           </span>
-          <span *ngIf="learningObject.contributors.length-2 > 0 "><span tip="{{ getContributors() }}">, +{{ learningObject.contributors.length - 2 }} more</span></span>
+          <span *ngIf="learningObject.contributors.length-2 > 0 ">
+            <span class="toolTip" tip="{{ getContributors() }}" >, and {{ learningObject.contributors.length - 2 }} other<span *ngIf="learningObject.contributors.length-2 > 2">s</span>
```

Comment:
`<span *ngIf="learningObject.contributors.length-2 > 2">s</span>` means that the s will only render if we have 3 or more, 2 or more won't render it. For example `[user1, user2]` has 2 users, but since 2 !> 2, we'd get `and 2 other`